src/applications/personalization/profile/components/personal-information/ConfirmRemoveModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from '@department-of-veterans-affairs/component-library/Modal';
+import { toLower } from 'lodash';
+import LoadingButton from '~/platform/site-wide/loading-button/LoadingButton';
+import { FIELD_NAMES } from '@@vap-svc/constants';
+import VAPServiceEditModalErrorMessage from '~/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage';
+
+const ConfirmRemoveModal = ({
+  cancelAction,
+  deleteAction,
+  title,
+  fieldName,
+  isEnrolledInVAHealthCare,
+  isLoading,
+  isVisible,
+  onHide,
+  error,
+}) => {
+  return (
+    <Modal
+      title={`Are you sure?`}
+      status="warning"
+      visible={isVisible}
+      onClose={onHide}
+    >
+      <div>
+        This will delete your {toLower(title)} across these VA benefits and
+        services:
+      </div>
+      <ul>
+        {isEnrolledInVAHealthCare && (
+          <li>
+            VA health care (including prescriptions, appointment reminders, lab
+            and test results, and communications from your VA medical center)
+          </li>
+        )}
+        <li>Disability compensation</li>
+        <li>Pension benefits</li>
+        <li>Claims and appeals</li>
+        <li>Veteran Readiness and Employment (VR&E)</li>
+        {fieldName !== FIELD_NAMES.EMAIL ||
+        fieldName !== FIELD_NAMES.MOBILE_PHONE ? (
```

Comment:
this will always eval to true, won't it?

Reply:
hmm... I would assume it would. Not sure why I did that. It looked like it was working right, but yeah that's not how that should read anyways. I will fix it